wherehows-dao/src/main/java/wherehows/dao/DaoFactory.java

```diff
@@ -24,4 +28,11 @@ public DatasetsDao getDatasetsDao() {
     return datasetsDao;
   }
 
```

Comment:
Sorry one more thing, add getDatasetClassificationDao() function here and use it in GobblinTrackingAuditProcessor.java